pkg/oc/cli/cmd/wrappers.go

```diff
@@ -788,3 +789,39 @@ func NewCmdAuth(fullName string, f *clientcmd.Factory, out, errout io.Writer) *c
 func NewCmdPlugin(fullName string, f *clientcmd.Factory, in io.Reader, out, errout io.Writer) *cobra.Command {
 	return kcmd.NewCmdPlugin(f, in, out, errout)
 }
+
+var (
+	apiresourcesExample = templates.Examples(`
+		# Print the supported API Resources
+		%[1]s api-resources
+
+		# Print the supported API Resources with more information
+		%[1]s api-resources -o wide
+
+		# Print the supported namespaced resources
+		%[1]s api-resources --namespaced=true
+
+		# Print the supported non-namespaced resources
+		%[1]s api-resources --namespaced=false
+
+		# Print the supported API Resources with specific APIGroup
+		%[1]s api-resources --api-group=extensions`)
+)
+
+func NewCmdApiVersions(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	cmd := kcmd.NewCmdApiVersions(f, out)
+	cmd.Example = fmt.Sprintf(apiresourcesExample, fullName)
```

Comment:
why bother replacing the example?

Reply:
Because the kube version uses kubectl in examples.